frontend-beeyond/src/app/core/services/backend-api.service.ts

```diff
@@ -9,71 +8,87 @@ import { TemplateApplication } from 'src/app/shared/models/template.application.
 import { Namespace } from 'src/app/shared/models/namespace.model';
 import { User } from 'src/app/shared/models/user.model';
 import { NamespaceSave } from 'src/app/shared/models/namespace-save.model';
+import { ConfigService } from './config.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class BackendApiService {
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, private configService: ConfigService) {}
```

Comment:
I would personally use a private member variable instead of `this.configService.config.backendApiUrl` all the time but it's just a matter of taste.